test/performance/latency_test.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 func TestTimeToServeLatency(t *testing.T) {
-	perfClients, err := Setup(t.Logf, true)
+	perfClients, err := Setup(t.Logf, t.Name(), true, false)
```

Comment:
Deja vu: are we going to make these constants?  This is really hard to read.

Reply:
Yes. I plan to do this in a later PR. I want to get this in first

Reply:
Updated with the const